Hello World String

```diff
@@ -0,0 +1,9 @@
+import java.util.*;
```

Comment:
This pull request do not contains files in specific folders. Please fix it and reopen